homeassistant/components/vicare/__init__.py

```diff
@@ -87,20 +67,25 @@ def setup_vicare_api(hass: HomeAssistant, entry: ConfigEntry) -> None:
     vicare_api = vicare_login(hass, entry.data)
 
     device_config_list = get_supported_devices(vicare_api.devices)
+    if number_of_devices := len(device_config_list) > 1:
```

Comment:
number_of_devices = `true`


```suggestion
    number_of_devices = len(device_config_list)
    if number_of_devices > 1:
```

Reply:
You need parentheses because the walrus operator has very low priority:
```suggestion
    if (number_of_devices := len(device_config_list)) > 1:
```